### PR TITLE
Convert OLM Views to Tables

### PR DESCRIPTION
**Pull request by alecmerdler**
### Description

Switches components in `package-manifest.tsx` and `k8s-resource.tsx` to use `<Table>`. Also cleans up some old code.

Addresses https://jira.coreos.com/browse/CONSOLE-1586

### CODE REVIEW

File: frontend/public/components/operator-lifecycle-manager/package-manifest.tsx [hunk 1]
```diff
@@ -85,8 +99,8 @@ export const PackageManifestList = requireOperatorGroup((props: PackageManifestL
               .filter(og => _.isEmpty(props.namespace) || og.metadata.namespace === props.namespace)
               .some(og => supports(installModesFor(rowProps.obj)(defaultChannelFor(rowProps.obj)))(og))}
           defaultNS={_.get(props.operatorGroup, 'data[0].metadata.namespace')} />}
-        label="Package Manifests"
-        EmptyMsg={() => <MsgBox title="No PackageManifests Found" detail="The catalog author has not added any packages." />} />
+        EmptyMsg={() => <MsgBox title="No PackageManifests Found" detail="The catalog author has not added any packages." />}
+        virtualize />
```
Comment (spadgett) [inline frontend/public/components/operator-lifecycle-manager/package-manifest.tsx]: Previously we disabled virtualization in this table. Is it safe to add it back?

Comment (alecmerdler) [reply frontend/public/components/operator-lifecycle-manager/package-manifest.tsx]: Should be. This view will be removed soon anyway.

**Pull request merged**